app/services/scm/delete_managed_repository_service.rb

```diff
@@ -35,14 +35,17 @@
   # Registers an asynchronous job to delete the repository on disk.
   #
   def call
-    if repository.managed?
-      delete_repository
+    return false unless repository.managed?
+
+    if repository.class.manages_remote?
+      Scm::DeleteRemoteRepositoryJob.new(repository).perform
+      true
```

Comment:
I just wanted the service be responsible to keep the result of `.call` a boolean instead of whatever the job returns. It should not make a difference in the end, but i prefer being more explicit here.